src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1191,6 +1190,34 @@ int MachCallNativeNode::ret_addr_offset() {
   return -1;
 }
 
+// RVC: With RVC a call may get 2-byte aligned.
+//   The offset encoding in jal ranges bits [12, 31], which could span the cache line.
```

Comment:
Maybe rephrasing like this:
//   The offset encoding in jal ranges bits [12, 31] could span the cache line.

Reply:
done. Thanks for polishing the sentense.